providers/image-service.ts

```diff
@@ -49,4 +62,91 @@ export class ImageService {
   getSanitizedImageBackground(url: string) {
     return this.sanitizer.bypassSecurityTrustStyle(`url(${url})`);
   }
+
+  /**
+   * Get image from the camera and just return the image data without uploading.
+   * @param direction The direction for the camera (`front` or `back`)
+   */
+  getImageFromCamera(direction?: string, width?: number, height?: number){
+    return this.loaderService.showLoader()
+    .switchMap(obs => Rx.Observable.of(this.app.isMobilePlatform()))
+    .flatMap(isMobile => {
+        this.loaderService.dismissLoader(); // Dismiss loader before opening Camera activity to avoid losing reference
+
+        if(isMobile) {
+            const options: CameraOptions = {
+                quality: 25,
+                destinationType: this.camera.DestinationType.DATA_URL,
+                encodingType: this.camera.EncodingType.JPEG,
+                mediaType: this.camera.MediaType.PICTURE,
+                correctOrientation: true
+              };
+
+              if(direction && direction === 'front') options['cameraDirection'] = this.camera.Direction.FRONT;
+              if(width && width > 0) options['targetWidth'] = width;
+              if(height && height > 0) options['targetHeight'] = height;
```

Comment:
same remark on convention, don't u prefer accessing the object variable directly instead of using the key string value ?

Reply:
Yep, man. It is kinda of a convention for me. I'm used to access an object that way when I'm adding a new property to a given object. Using the pattern that we defined now on.